src/helpers/folder_helpers.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Helper methods for dealing with directories.
+
+
+# Changes into the given directory
+function change_directory {
+  local directory=$1
+  pwd
+  run_git_command "cd $directory"
```

Comment:
Yeah, we have some technical drift here. I have opened #493 to address this separately, because it will result in a larger shotgun refactor.

I'd like to keep printing this command to the user, and verifying it in the commands table in Cucumber, because it is a relevant operation that both the user and the feature specs should be aware of.